plasma/limbo-priority-mod.md

```diff
@@ -0,0 +1,58 @@
+
+This has motivated the construction of alternative mechanisms which provide the same guarantees. We introduce a design whereby any party may attempt to set the exit priority of a transaction. This means that the transaction would now be processed earlier in the exit priority queue. This attempt is challengeable if the new priority would be greater than the priority of any of the transaction's inputs, if any of the inputs have been double-spent, or if the transaction's new priority would be lower than it's current priority. 
+
+The effect of this design is that an in-flight transaction that may or may not be included in the Plasma chain (a transaction "in limbo") can now be guaranteed to be included. Furthermore, if all of a transactions inputs were created at a time when the chain was valid, then the transaction will be successfully processed before any invalid transactions are processed.
+
+## Construction
+
+A user may attempt to set the priority of a transaction by submitting a signed transaction and a block number to the root chain, subject to a challenge period. This action is bonded, and the priority of deposit ("out of nowhere") transactions cannot be modified. If no successful challenge occurs during the challenge period, then the transaction will have a priority as if it were the added to the end of the given block. 
+
+A priority modification can be challenged in four ways (higher priority = would be exited sooner, lower priority = would be exited later):
+
+1. Someone shows that the specified transaction was included in a block, and that its original priority is higher than the new priority. This prevents someone from decreasing a transaction's priority. 
+2. Someone shows that any of the transaction's inputs would have a lower priority than the transaction. This prevents someone from moving a transaction "earlier" than any of the transaction's outputs.
+3. Someone requests that someone prove some input to the transaction has a higher priority than new priority. This is necessary if blocks are being withheld and a challenge of type (2) is not directly possible.
+4. Someone shows that there exists another spend from any of the transaction's inputs. This spend must not be the same as the original transaction.
+
+A challenge of type (1) or (2) blocks the modification and slashes the submitter's bond. 
+
+A challenge of type (3) or (4) must be bonded, and are subject to a response.
+
+### Type (3) Response
+
+Anyone may respond to a type (3) challenge by revealing that the requested input has been included in the chain such that it has a lower priority than the new priority. A valid response would slash the challenger's bond. A failure to respond would slash the original bond and block the modification.
```

Comment:
Here response proves that inputs have higher priority than new priority of tx. "Lower priority" is used.

Reply:
Thanks, changed.